nsdl-c/sn_nsdl_lib.h

```diff
@@ -437,51 +435,47 @@ extern int8_t sn_nsdl_put_resource(struct nsdl_s *handle, sn_nsdl_dynamic_resour
 extern int8_t sn_nsdl_pop_resource(struct nsdl_s *handle, sn_nsdl_dynamic_resource_parameters_s *res);
 
 /**
- * \fn extern int8_t sn_nsdl_delete_resource(struct nsdl_s *handle, uint8_t pathlen, uint8_t *path)
+ * \fn extern int8_t sn_nsdl_delete_resource(struct nsdl_s *handle, char *path)
```

Comment:
This path argument here and most other places could/should now be "const char*".